frontend/src/screens/learner/details.tsx

```diff
@@ -178,14 +185,28 @@ export const StudyDetails: React.FC<{ studies: ParticipantStudy[] }> = ({ studie
     )
 }
 
+const renderResearcherSection = (study: Study | ParticipantStudy) => {
```

Comment:
Good job! Only comment here is instead of having this be a function and calling it in the render method on line 232, you can just make this a "component" like so:

```suggestion
const ResearcherSection = (study: Study | ParticipantStudy) => {
```

and then when using this:
```
<s>{renderResearcherSection(study)}</s>
<ResearcherSection study={study} />
```